trl/trainer/ppo_trainer.py

```diff
@@ -993,6 +985,14 @@ def record_step_stats(self, kl_coef: float, **data):
 
         mean_non_score_reward = masked_mean(data["non_score_reward"], mask)
 
+        if mean_kl.item() < 0.0:
+            # warn users
+            warnings.warn(
+                f"KL divergence is starting to become negative: {mean_kl.item()} - this might be a precursor for failed training."
```

Comment:
for better readability :) 
```suggestion
                f"KL divergence is starting to become negative: {mean_kl.item():.2f} - this might be a precursor for failed training."
```